package.json

```diff
@@ -35,6 +35,15 @@
       "js",
       "vue"
     ],
+    "collectCoverage": true,
+    "collectCoverageFrom": [
```

Comment:
I think we should show something to ignore tests from coverage results instead:

```js
"coveragePathIgnorePatterns": [
  "<rootDir>/test/*.js"
]
```

Reply:
What's the reasoning?

Reply:
Mmh, I'm not sure why I said it that way.
I actually think It's better to use collectCoverageFrom but it does need to be updated (`src/**/*.{js,vue}`) because otherwise it will take every file. No need to ignore `node_modules` if no `coveragePathIgnorePatterns` is provide as it contains it by default but we may want to ignore `main.js` or `App.vue` (or both) and it can also serve as an example